HEN_HOUSE/src/EEMF_macros.mortran

```diff
@@ -0,0 +1,2397 @@
+
+
+" The next line is exactly 80 columns wide. Wider is not recognized by Mortran"
+"234567890123456789012345678901234567890123456789012345678901234567890123456789"
+
+"******************************************************************************"
+"**************************     EM Field Macros     ***************************"
+"******************************************************************************"
+" This code implements charged particle in constant electric/magnetic fields.  "
+" In the current implementation, the magnetic field has been tested and        "
+" validated (Malkov, V.N. and Rogers, D.W.O., 2016. Charged particle transport "
+" in magnetic fields in EGSnrc. Medical Physics, 43(7), pp.4447-4458.)"
+"******************************************************************************"
+"******************************************************************************"
+
+"******************************************************************************"
+"************************** Algorithm parameters ******************************"
+"******************************************************************************"
+"Step size control for upper bound on 1PI for magnetic field simulations"
+"Below the stepsize associated with this delta_u value 1PI is used"
+" Above this, 3PI is applied"
+REPLACE{$MINS_ANG}WITH{0.05};
+
+"Default delta_u value if input file definitions are missing"
+REPLACE{$deltaUDefault}WITH{0.02};
+
+"Maximum energy loss due to electric field (step-size restriction)"
+REPLACE{$MAXS_eLoss}WITH{0.1};
+
+" ******** algorithm options  ******** "
+"Integration order selection for PRESTA"
+"   Value can be 1, 2 or 3"
+REPLACE{$IntegrationAlg}WITH{3};
+"Single scattering algorithm selection"
+" Value can either be 1 or 2:"
+" 1: A 1 point integration of the effect of the EM field"
+" 2: Analytical solution to propogration of particle in magnetic field."
+"    (defaults back to 1 point integration for electric fields "
+REPLACE{$ssFieldAlg}WITH{2};
+
+"******** BCA variables ******** "
+" If using BCA set these parameter "
+REPLACE{$BCA_lin_buffer}WITH{min(1e-6,skindepth)};
+REPLACE{$BCA_lin_dist}WITH{min(1e-5,skindepth)};
+
+
+" ****************** BOOLEANS *****************"
+REPLACE{$orderBoundary}WITH{.true.}; "selecting 1PI v 3PI so no boundary errors"
+REPLACE{$orderAdaptive}WITH{.true.}; "Selecting 1PI v 3PI for B for efficiency"
+REPLACE{$duAdaptive}WITH{.true.}; "for B, scaling the step size to a 1.5 T"
+REPLACE{$energyLossViaPotential}WITH{.false.};"Boolean for using potential"
+      "for energy loss. "
+" Determine if output at the begining of the simulation is needed: "
+REPLACE{$EMInternalOutput}WITH{.true.};
+
+
+"******************************** CONSTANTS ***********************************"
+REPLACE{$EM_MACROS_ACTIVE}WITH{.true.};
+REPLACE{$SL}WITH{299792458.0}; "speed of light"
+REPLACE{$ERM}WITH{0.510998910}; "electron rest energy in MeV"
```

Comment:
Why a different value than PRM from egsnrc.macros?

Reply:
I think we can address these constants after we have merged into master for the release. The main question is - if we update them for consistency with the rest of the code, will it affect the EMF algorithm? E.g. are there any functional fits that should be recalculated with the new values?

Reply:
Ok, let's leave the hard-coded values in for now, but please post an issue to remind ourselves to update with the EGSnrc-wide constants.